centreon/tests/e2e/cypress/e2e/Dashboards/05-dashboard-check-permissions.feature

```diff
@@ -1,8 +1,10 @@
+@REQ_MON-22172
 Feature: Checking dashboard permissions
```

Comment:
@tuntoja @paul-oureib should it be https://centreon.atlassian.net/browse/MON-18516 ?